test/indexer/data/aee55415-d128-4b30-9644-e6b2742fa32b.files.results.json

```diff
@@ -123,12 +123,143 @@
                                     }
                                 ],
                                 "project": {
+                                    "project_title": "Melanoma infiltration of stromal and immune cells",
```

Comment:
Is this file actually used?

Reply:
It's used in `test_index_correctness` in `test_hca_indexer.py`